pkg/controllers/generic/export_syncer.go

```diff
@@ -46,7 +46,17 @@ func CreateExporters(ctx *context.ControllerContext, config *config.Config) erro
 		}
 	}
 
-	for _, exportConfig := range config.Exports {
+	for _, exportConfig := range exporterConfig.Exports {
+		reversePatches := []*config.Patch{
+			{
+				Operation: config.PatchTypeCopyFromObject,
+				FromPath:  "status",
+				Path:      "status",
+			},
+		}
+		reversePatches = append(reversePatches, exportConfig.ReversePatches...)
+		exportConfig.ReversePatches = reversePatches
```

Comment:
What are the advantages or use cases of the implicit status reverse patch? Besides shorter config of course. :)

Reply:
its mostly shorter config, the question would be why not to include it, as you probably always want it